<ui version="4.0" >
 <class>TimeControlWidget</class>
 <widget class="QWidget" name="TimeControlWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="button_reconstruction_increment" >
     <property name="toolTip" >
      <string>Step backwards one increment  Ctrl+Shift+I</string>
     </property>
     <property name="icon" >
      <iconset resource="../qt-resources/qt_widgets.qrc" >:/gnome_seek_backward_22.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRepeat" >
      <bool>true</bool>
     </property>
     <property name="autoRepeatInterval" >
      <number>200</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_reconstruction_decrement" >
     <property name="toolTip" >
      <string>Step forwards one increment  Ctrl+I</string>
     </property>
     <property name="icon" >
      <iconset resource="../qt-resources/qt_widgets.qrc" >:/gnome_seek_forward_22.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRepeat" >
      <bool>true</bool>
     </property>
     <property name="autoRepeatInterval" >
      <number>200</number>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_time" >
     <property name="text" >
      <string>Time:</string>
     </property>
     <property name="buddy" >
      <cstring>spinbox_current_time</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinbox_current_time" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip" >
      <string>Set reconstruction time  Ctrl+T</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum" >
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_reconstruction_unit" >
     <property name="text" >
      <string>Ma </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt-resources/qt_widgets.qrc" />
 </resources>
 <connections/>
</ui>
